W-13260932: add APIkit for REST & SOAP overviews (#131)
mulesoft/docs-apikit: modules/ROOT/pages/apikit-for-soap.adoc

= APIkit for SOAP
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> TBD

Reviewer: Is this removal OK?

Reviewer: Ahh, this is basically `modules/ROOT/pages/apikit-4-for-soap.adoc`

include::_attributes.adoc[]
endif::[]

APIkit for SOAP is a dedicated framework designed to simplify the development and integration of SOAP-based APIs. SOAP (Simple Object Access Protocol) is a messaging protocol used for exchanging structured information between web services. With APIkit for SOAP, you gain a powerful set of tools and features that streamline SOAP API development, ensuring seamless integration with your existing SOAP infrastructure.

By leveraging APIkit for SOAP, you can experience numerous benefits in your SOAP API development process. It simplifies the handling of SOAP message formats and headers, enabling you to focus on the core logic of your APIs. This streamlines the development process, making it more efficient and reliable. Additionally, APIkit for SOAP automates the binding to WSDL (Web Services Description Language), reducing manual configuration efforts and ensuring accuracy.

> Features:
> 
> * *Automatic binding to WSDL*: APIkit for SOAP automatically binds to your WSDL definitions, saving you time and effort in configuring your APIs.
> 
> * *Efficient handling of SOAP messages*: With APIkit for SOAP, you can seamlessly handle SOAP request and response messages, enabling smooth communication between SOAP-based services.
> 
> * *Support for data transformations*: The framework provides robust support for data transformations, enabling you to easily manipulate and format data within SOAP APIs.
> 
> By placing emphasis on SOAP/WSDL contracts, APIkit for SOAP enables you to define and expose SOAP APIs with ease. Its capabilities in managing SOAP message formats and headers ensure seamless integration with SOAP infrastructure. By leveraging the features of APIkit for SOAP, you can streamline the SOAP API development process, resulting in more efficient and reliable SOAP-based APIs.
> 
> With APIkit for SOAP, you can efficiently create robust and reliable SOAP-based APIs, accelerating your development workflow and delivering enhanced integration capabilities.

Reviewer 2: All of this sound pretty similar to what is mentioned above, almost with the same words but in different order. In my opinion I would get rid of it. Maybe keep something that was not mentioned or summarize it.
```suggestion
With APIkit for SOAP, you can efficiently implement robust and reliable SOAP-based APIs, accelerating your development workflow and delivering enhanced integration capabilities.
```

Reviewer 3: Thanks!
